config/config.go

```diff
@@ -1423,24 +1423,38 @@ func GetFilePath(file string) (string, error) {
 		filepath.Join(newDir, EncryptedConfigFile),
 	}
 
-	// First upgrade the old dir config file if it exists to the corresponding new one
+	// First upgrade the old dir config file if it exists to the corresponding
+	// new one
 	for x := range oldDirs {
 		_, err := os.Stat(oldDirs[x])
 		if os.IsNotExist(err) {
 			continue
 		}
+		_, err = os.Stat(newDirs[x])
+		if !os.IsNotExist(err) {
+			log.Warnf(log.ConfigMgr,
+				"Warning: config.json file found in root dir and gct dir; cannot overwrite, defaulting to gct dir config.json at %s",
```

Comment:
Can drop the "warning" as you're already using Warnf, we used warning prefixes in the past when we didn't have a fancy logger